.github/workflows/refresh_voc_and_counts.yml

```diff
@@ -0,0 +1,44 @@
+name: Refresh vocabulary and counts tables
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'  # weekly, sunday 12am
```

Comment:
@Sigfried I set this to run Sundays at 12am. Let me know if you can think of a better time.